examples/3d/3d_shapes.rs

```diff
@@ -70,6 +70,7 @@ fn setup(
             shadows_enabled: true,
             intensity: 10_000_000.,
             range: 100.0,
+            shadow_depth_bias: 0.4,
```

Comment:
I don’t remember the default but we could change it if needed.

Reply:
It was 0.02. I changed the default to 0.08, which was the lowest value that made the shadow acne go away in those demos (I think anyway; it's hard to tell).